interfaces/json/session.ts

```diff
@@ -8,6 +8,22 @@ import {
 import { StatutAnimationCollectiveJson } from 'interfaces/json/evenement'
 import { minutesEntreDeuxDates } from 'utils/date'
 
+type InscriptionSessionJson = {
+  idJeune: string
+  nom: string
+  prenom: string
+  statut: string
+}
+export type SessionMiloBeneficiaireJson = {
```

Comment:
```suggestion
export type SessionMiloBeneficiaireAvecInscriptionsJson = {
export type SessionMiloBeneficiaireAgendaConseillerJson = {

```
un peu long mais pour éviter le conflit avec l'équivalent de la fichebeneficiaire. ou de mon côté mettre le suffixe FicheBeneficiaire...